postgres/assets/configuration/spec.yaml

```diff
@@ -631,9 +649,37 @@ files:
           value:
             type: string
             example: my-postgres-database.database.windows.net
+        - name: managed_authentication
+          description: |
+            Configuration section used for Azure AD Authentication.
+        
+            This supports using System or User assigned managed identities.
+            
+            For more information on Managed Identities, see the Azure docs
+            https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview
+
+            To enable Azure AD Authentication, set `azure.managed_authentication.enabled` to `true`.
+            If `azure.managed_authentication.enabled` is set, then the `username` and `password` fields will be ignored.
+            `azure.managed_authentication.client_id` is required to enable Azure AD Authentication.
+
+            For more information on scopes, see the Azure docs
```

Comment:
we might want to link to our own docs here as well https://docs.datadoghq.com/database_monitoring/guide/managed_authentication/

since they have to do a bunch of extra stuff to set this up

Reply:
oh also, will need to update those docs once this is released

Reply:
I will make a ticket for myself to update the doc once this is released